docs/changelog.md

```diff
@@ -0,0 +1,1157 @@
+# Changelog
```

Comment:
What is the difference between the root CHANGELOG.md and this file?

Reply:
Yes docs/changelog.md has to be in gitignore

Reply:
The docs.changelog.md file needs to be deleted from this PR